cupsctl, lpq, lprm, lpadmin, lpoptions: add pages (#11745)
tldr-pages/tldr: pages/common/lpq.md

# lpq

> Show printer queue status.
> More information: <https://www.cups.org/doc/man-lpq.html>.

- Show the queued jobs of the default destination:

`lpq`

> - Show the queued jobs of [a]ll printers enforcing [E]ncryption:

Reviewer: ```suggestion
- Show the queued jobs of all printers enforcing encryption:
```

`lpq -a -E`

> - Show the queued jobs in a long format:

Author: Eu já tinha removido o mnemônico depois da revisão do @kbdharun 😅

> - Show the queued jobs in a [l]ong format:

Reviewer: ```suggestion
- Show the queued jobs in a long format:
```
